generated/xamarin-partition-based-sync/realm-todo-dotnet/App.xaml.cs

```diff
@@ -34,7 +35,7 @@ protected override void OnStart()
                 NavigationPage.SetHasBackButton(navPage, false);
                 MainPage = navPage;
             }
-            catch (Exception e)
+            catch (Exception ex)
             {
                 // A NullReferenceException occurs if:
```

Comment:
Given that here you are capturing all exceptions, and not only the `NullReference` ones, I would change the text to say `An exception can occur if`. 
It could be that different exceptions are coming from that piece of code anyways, so we stay more on the general side